js/jquery.emojiarea.js

```diff
@@ -807,6 +854,8 @@
 
         // Signifies if emoji picker is visible
         window.emojiPickerStatus.isPickerVisible = false;
+        // This is the instance of the emoji picker that is currently visible.
+        window.emojiPickerStatus.visibleEmojiPickerInstance = null;
```

Comment:
Would it be better to use updateGlobalEmojiPickerVisiblity to keep the visibility code together?